src/methods/call.js

```diff
@@ -305,7 +305,7 @@ export function OrdinaryCallEvaluateBody(realm: Realm, F: FunctionValue, argumen
     }
     if (c instanceof JoinedAbruptCompletions) {
       if (e !== undefined) realm.applyEffects(e);
-      throw AbstractValue.createIntrospectionErrorThrowCompletion(c.joinCondition);
+      return c;
```

Comment:
In the non partial evaluation case the introspection error now gets created later on.